testsuite/MDAnalysisTests/coordinates/test_trc.py

```diff
@@ -0,0 +1,189 @@
+from MDAnalysisTests.datafiles import TRC_CLUSTER_VAC
+from MDAnalysisTests.datafiles import TRC_PDB_SOLV, TRC_TRAJ_SOLV
+
+
+class TestTRCReaderVacuumBox:
+    @pytest.fixture(scope='class')
+    def TRC_U(self):
+        return mda.Universe(TRC_PDB_VAC, [TRC_TRAJ1_VAC, TRC_TRAJ2_VAC],
+                            continuous=True)
+
+    def test_initial_frame_is_0(self, TRC_U):
+        assert_equal(TRC_U.trajectory.ts.frame, 0,
+                     "The initial frame is not 0 but {0}".format(
+                         TRC_U.trajectory.ts.frame))
+
+    def test_trc_positions(self, TRC_U):
+        # first frame first particle
+        TRC_U.trajectory[0]
+        assert_allclose(TRC_U.atoms.positions[0],
+                        [2.19782507, 24.65064345, 29.39783426])
+        # fith frame first particle
+        TRC_U.trajectory[4]
+        assert_allclose(TRC_U.atoms.positions[0],
+                        [0.37026654, 22.78805010, 3.69695262])
+
+    def test_trc_dimensions(self, TRC_U):
+        assert TRC_U.trajectory[0].dimensions is None
+
+    def test_trc_n_frames(self, TRC_U):
+        assert len(TRC_U.trajectory) == 6
+        assert (TRC_U.trajectory.n_frames) == 6
+
+    def test_trc_n_atoms(self, TRC_U):
+        assert (TRC_U.trajectory.n_atoms) == 73
+
+    def test_trc_frame(self, TRC_U):
+        assert TRC_U.trajectory[0].frame == 0
+        assert TRC_U.trajectory[4].frame == 4
+
+    def test_trc_time(self, TRC_U):
+        assert TRC_U.trajectory[0].time == 0
+        assert TRC_U.trajectory[4].time == 80
+
+    def test_trc_dt(self, TRC_U):
+        time_array = np.array([ts.time for ts in TRC_U.trajectory])
+        assert_equal(time_array, np.arange(6)*20)
+
+        dt_array = np.diff(time_array)
+        assert_equal(dt_array, np.full(5, 20))
+
+    def test_trc_data_step(self, TRC_U):
+        assert TRC_U.trajectory[0].data['step'] == 0
+        assert TRC_U.trajectory[4].data['step'] == 10000
+
+    def test_periodic(self, TRC_U):
+        assert_equal(TRC_U.trajectory.periodic, False)
+
+    def test_rewind(self, TRC_U):
+        TRC_U.trajectory[0]
+        trc = TRC_U.trajectory
+        trc.next()
+        trc.next()
+        trc.next()
+        trc.next()
+        assert_equal(trc.ts.frame, 4,
+                     "trajectory.next() did not forward to frameindex 4")
+        trc.rewind()
+        assert_equal(trc.ts.frame, 0, "trajectory.rewind() failed "
+                                      "to rewind to first frame")
+
+        assert np.any(TRC_U.atoms.positions != 0), "The atom positions " \
+                                                   "are not populated"
+
+    def test_random_access(self, TRC_U):
+        TRC_U.trajectory[0]
+        pos0 = TRC_U.atoms[0].position
+        TRC_U.trajectory.next()
+        TRC_U.trajectory.next()
+        pos2 = TRC_U.atoms[0].position
+
+        TRC_U.trajectory[0]
+
+        assert_equal(TRC_U.atoms[0].position, pos0)
```

Comment:
For any floating point number comparisons such as coordinates, use [assert_allclose](https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_allclose.html)
```python
assert_allclose(value, reference)
```

Reply:
In principle I agree, although it would bother me in this specific case if a numerically different value is returned.

I will still change it as you recommended.